vuepress/docs/next/docs/getting-started/README.md

```diff
@@ -2,9 +2,28 @@
 sidebarDepth: 2
 ---
 
-# Get Started with Entando in 3 Easy Steps
+# Get Started with Entando
 
-New to Kubernetes, hypervisors, and Helm charts?
+You have two options for getting started with Entando.
+
+1. Use the Entando CLI to seamlessly install a Kubernetes, prepare it with Entando resources, and deploy Entando to it.   
+    1. Install Multipass
+
+    ``` http request
+    https://multipass.run/#install
+   ```
+
+    2. Install Entando
+
+    ```sh
+    curl "https://raw.githubusercontent.com/entando/entando-cli/develop/hr/auto" |\
+    ENTANDO_RELEASE="v6.3.0-sprint12" ENTANDO_CLI_VERSION="develop" bash
+   ```
+
+   3. The progress of the install will be displayed on the console and can take 10 minutes or so depending on your network bandwidth. Once complete, the installer will give you the URL to access the <span>Entando App Builder</span>, e.g. `https://quickstart-entando.192.168.64.33.nip.io/app-builder/`. Login with username:`admin`, password: `adminadmin`.
```

Comment:
I wonder if "...depending on the time to download the docker images" is better than "network bandwidth"

Reply:
I like that better. I'll update it when I get back to it.

Reply:
Updated